fuzzers/qemu_systemmode/src/fuzzer_classic.rs

```diff
@@ -84,9 +84,21 @@ pub fn fuzz() {
 
     let mut run_client = |state: Option<_>, mut mgr, _core_id| {
         // Initialize QEMU
-        let args: Vec<String> = env::args().collect();
-        let env: Vec<(String, String)> = env::vars().collect();
-        let qemu = Qemu::init(&args, &env).unwrap();
+        let qemu = Qemu::builder()
+            .machine("mps2-an385")
+            .monitor(qemu_config::Monitor::null)
+            .kernel("target/classic/example.elf")
+            .serial(qemu_config::Serial::null)
+            .no_graphic(true)
+            .snapshot(true)
+            .drives(vec![qemu_config::Drive::builder()
```

Comment:
There's probably a way for this to take a `&[]` instead of a vec, that'd make this look even nicer, but otherwise it now looks really good! <3